### PR TITLE
Removed <> from ElementsShouldBeAtLeast and ConditionAndGroupGenericParameterTypeShouldBeTheSame

### PR DESCRIPTION
**Pull request by himadri-mandal**
#### Check List:
* Fixes: Removed <> from ElementsShouldBeAtLeast, ConditionAndGroupGenericParameterTypeShouldBeTheSame,
                                           ElementsShouldBeAtLeast, ElementsShouldBeAtMost, ElementsShouldBeExactly  (see #2091)
* Unit tests : YES
* Javadoc with a code example (on API only) : NA

### CODE REVIEW

File: src/main/java/org/assertj/core/error/ConditionAndGroupGenericParameterTypeShouldBeTheSame.java [hunk 1]
```diff
@@ -24,7 +24,7 @@
 public class ConditionAndGroupGenericParameterTypeShouldBeTheSame extends BasicErrorMessageFactory {
 
   public ConditionAndGroupGenericParameterTypeShouldBeTheSame(Object actual, Condition<?> condition) {
-    super("%nExpecting: <%s> have the same generic type as condition <%s>", actual, condition);
+    super("%nExpecting: %s have the same generic type as condition %s", actual, condition);
```
Comment (joel-costigliola) [inline src/main/java/org/assertj/core/error/ConditionAndGroupGenericParameterTypeShouldBeTheSame.java:27]: I realize there is a missing `to` here, should be `Expecting: %s to have`

File: src/main/java/org/assertj/core/error/ElementsShouldBe.java [hunk 2]
```diff
@@ -35,7 +35,7 @@ public static ErrorMessageFactory elementsShouldBe(Object actual, Object notSati
   }
 
   private ElementsShouldBe(Object actual, Object notSatisfies, Condition<?> condition) {
-    super("%nExpecting elements:%n<%s>%n of %n<%s>%n to be <%s>", notSatisfies, actual, condition);
+    super("%nExpecting elements:%n%s%n of %n%s%n to be %s", notSatisfies, actual, condition);
```
Comment (joel-costigliola) [inline src/main/java/org/assertj/core/error/ElementsShouldBe.java:38]: Fix the indentation to be like `"%nExpecting elements:%n  %s%nof%n  %s%nto be %s"`

Comment (joel-costigliola) [reply src/main/java/org/assertj/core/error/ElementsShouldBe.java:38]: apply the same indentation logic to other ElementsShouldBeXXX classes

File: src/main/java/org/assertj/core/error/ConditionAndGroupGenericParameterTypeShouldBeTheSame.java (hunk 1 above)
Comment (himadri-mandal) [reply src/main/java/org/assertj/core/error/ConditionAndGroupGenericParameterTypeShouldBeTheSame.java:27]: Yes, I am adding them.

File: src/main/java/org/assertj/core/error/ElementsShouldBe.java (hunk 2 above)
Comment (himadri-mandal) [reply src/main/java/org/assertj/core/error/ElementsShouldBe.java:38]: I am adding them for all ElementsShouldBeXXX classes.

Comment (himadri-mandal) [reply src/main/java/org/assertj/core/error/ElementsShouldBe.java]: @joel-costigliola  Covered all ElementsShouldBeXXX classes with suggested indentation. Do you want a similar indentation for all ElementsShouldHaveXXX classes as well?

**Comment by joel-costigliola** [comment]
Integrated thanks @himadri-mandal!
I have added ElementsShouldHave, used two spaces for indentation but you get all the credit :wink: .
One thing, it's better for us if you rebase your PR on main instead of merging main so that the history does not show branches but simple commits

**Pull request closed without merging**